Doodads/DoodadsPlugin.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using Doodads.Commands;
+using Terraria;
+using TerrariaApi.Server;
+using TShockAPI;
+
+namespace Doodads
+{
+	/// <summary>
+	/// The main plugin class should always be decorated with an ApiVersion attribute. The current API Version is 1.22
+	/// </summary>
+	[ApiVersion(1, 22)]
```

Comment:
Not something that needs changing in this PR seeing as this is an initial commit, but the current API version is 1.25.